mizar/common/common.py

```diff
@@ -289,12 +289,14 @@ def kube_list_namespaces_by_labels(core_api, label_dict):
     except:
         return None
 
-def kube_list_pods_by_labels(core_api, label_dict):
+def kube_list_pods_by_labels(core_api, label_dict, namespace=None):
```

Comment:
what is the semantic meaning for none namespace? all namespaces?

Reply:
Yes, when not sending namespace value, it means searching from all namespaces.